src/nvim/popupmnu.c

```diff
@@ -36,8 +38,11 @@ static int pum_scrollbar;           // TRUE when scrollbar present
 static int pum_row;                 // top row of pum
 static int pum_col;                 // left column of pum
 
-static int pum_do_redraw = FALSE;   // do redraw anyway
+static bool pum_is_visible = false;
+static bool pum_do_redraw = false;   // do redraw anyway
```

Comment:
`pum_do_redraw` is only set before calls to `update_screen()` in `pum_set_selected()`, so that the pum is not drawn when updating the screen. Can't we just set `pum_is_visible = false` before calling `update_screen()` in `pum_set_selected()`  and remove the variable `pum_do_redraw` completly?